### DISCUSSION TITLE
sort components by storage location bug?

### DISCUSSION THREAD

**Discussion by jboeck** [Part-DB/Part-DB-server#269] [Q&A]
Hi, i think there's a bug sorting components by storage location or do i something wrong?

**Comment by jbtronics** [comment]
What behaviour would you expect and what happens actually?

**Reply by jboeck** [reply]
I can't select the 'Storage Location' column to sort. The hand appears in the other columns and I can sort but not there. The arrow for the sorting direction is also missing.

**Comment by Pyromane** [comment]
Here is a screen from the demo page:
![image](https://user-images.githubusercontent.com/5962901/231548137-e4d9977d-97cb-448f-8c28-0a351277064a.png)

**Comment by jbtronics** [comment]
Sorting is now possible. However, as a part can have multiple storage locations, the ordering can maybe lead to unexpected results, when a part has multiple part lots.
